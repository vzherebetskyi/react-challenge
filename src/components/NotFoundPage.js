import React from 'react';

const NotFoundPage = () => (
  <div className="page-container">
    <div className="page-wrapper" style={{ alignItems: 'center' }}>
      <div style={{ padding: '2rem' }}>
        <h1>Not Found</h1>
        <h3>Oops, this page is being developed or does not exist</h3>
      </div>
    </div>
  </div>
);

export default NotFoundPage;
