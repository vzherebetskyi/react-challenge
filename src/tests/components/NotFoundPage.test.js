import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '../../components/NotFoundPage';

test('should render NotFoundPage correctly', () => {
  const elem = renderer.create(<NotFoundPage />).toJSON();
  expect(elem).toMatchSnapshot();
});
