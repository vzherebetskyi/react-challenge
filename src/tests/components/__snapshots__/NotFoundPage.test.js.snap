// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="page-container"
>
  <div
    className="page-wrapper"
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <div
      style={
        Object {
          "padding": "2rem",
        }
      }
    >
      <h1>
        Not Found
      </h1>
      <h3>
        Oops, this page is being developed or does not exist
      </h3>
    </div>
  </div>
</div>
`;
